edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -512,6 +512,34 @@ Generation of groups is an organizer responsibility, but as a Delegate it is you
 
 For non-first rounds, it is a good practice to put all of the similarly skilled competitors in one group to ensure that they receive the same scrambles. Use the previous round’s results to determine which groups to place competitors.
 
+## Competition Cancellation and Rescheduling
+
+Once a competition is announced, it is the expectation and responsibility of the Delegates to commit to seeing the competition through on the original date announced. Competitors and their families will have made arrangements and financial commitments to attend on that day, and it is in everyone’s best interest to honor those commitments.
+
+A cancellation should only happen when there is no safe or feasible alternative. This is only an option in extenuating circumstances such as:
+
+- Extreme weather conditions/natural disasters
+- Political unrest or conflict
+- Venue facility failures (electrical, plumbing, structural), where another venue is not possible
+- General safety concerns with competitors traveling to or attending the competition
+- Force majeure
+
+Competitions may not be canceled for the following reasons:
+
+- Low registration numbers
+- Financial concerns
+- Delegate(s) suddenly unable to attend
```

Comment:
This also seems like it could be an extenuating circumstance. For example, in the most extreme case, the Delegate could die, haha.

Reply:
And they could be the only listed Delegate in a remote region where another Delegate is unavailable

Reply:
Oh, I guess the point below explains what to do in such a case, nvm

Reply:
I think I can resolve this then lol